eng/automation/dotnet-install.sh

```diff
@@ -0,0 +1,1568 @@
+#!/usr/bin/env bash
```

Comment:
We already have .net init script [here](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/Automation-Sdk-Init.ps1), we use it in the specs repo for HLC library automation. Can we reuse [this](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/Automation-Sdk-Init.ps1) script?

Reply:
This dotnet-install.sh is used by docker solution to generate 10-min package  The docker use Alpine 3.15.0, and the dotnet-install.sh downloaded from .Net website is not support this version, need a little update, so I add the updated version here.

Reply:
If possible we should try and avoid having this file commited to the repo. As I mentioned in my other comment we should just start with a base dotnet docker image (ideally the one in the root .devcontainer folder of the repo) to avoid needing to install anything.

Reply:
The docker solution is target for all SDK language. So they will choose one common linux image suitable for all SDKs.

Reply:
Why do we need a single docker file for all languages? Each language needs a custom set of tools to build and we will have to maintain a lot of scripts/steps to install those prerequisites. All the languages have a .devcontainer with the exact purpose of having a starting point of working in our repos. We should instead use those docker containers as the base image and it is OK to have a unique base image for each language. I think that is better than having a common docker image and then maintain a list of install steps for each. That list of install steps already needs to be maintained for the devcontainer and we should share those.

Reply:
This is a feature required from Peter and Laurent. We will provider such docker for all SDK language and provide centralized and friendly SDK experience for service team.

Reply:
@chunyu3 I agree with the request but I think we can and should use the devcontainers for each of the languages. We don't need to have a single docker image across all languages.

Reply:
OK. thanks, Wes. Since all-in-one docker have changed the base image to regular ubuntu, so the dotnet-install.sh does not need anymore. I will delete it.